plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -175,6 +175,9 @@ open class KtlintPlugin : Plugin<Project> {
             // Android option is available from ktlint 0.12.0
             runArgs.add("--android")
         }
+        if (extension.ruleSets.size > 0 && SemVer.parse(extension.version) >= SemVer(0, 20, 0)) {
```

Comment:
According to https://github.com/shyiko/ktlint/commit/cf27ae8096a3166bce44ec843e3765ff7edcfbbc#diff-fb157ca11660a4b329fbc3faf6c2ed5e `--ruleset` option is supported since initial release, so you don't need add `SemVer` check here.